protocol/src/domains/SqlDomain.ts

```diff
@@ -1,30 +1,19 @@
 import { DomainDef } from '@nodescript/protocomm';
 
-import { SqlModificationResult, SqlModificationResultSchema } from '../schema/SqlModificationResult.js';
-import { SqlQueryResult, SqlQueryResultSchema } from '../schema/SqlQueryResult.js';
+import { SqlQueryResultSchema } from '../schema/SqlQueryResult.js';
 
 export interface SqlDomain {
 
     connect(req: {
         connectionUrl: string;
     }): Promise<{}>;
 
-    executeDefinition(req: {
+    query(req: {
         connectionUrl: string;
-        definition: string;
+        query: string;
+        params?: any[];
     }): Promise<{}>;
```

Comment:
Should still return something right? Weird that TS didn't catch that 🤔

Reply:
Huh, yeah that's weird - oops! Thanks for catching it